### PR TITLE
Update getting-started-minikf.md

### PR DESCRIPTION
**Pull request by jbottum**

### CODE REVIEW

File: content/en/docs/started/workstation/getting-started-minikf.md [hunk 1]
```diff
@@ -3,17 +3,8 @@ title = "MiniKF"
...
-
-![MiniKF latest
-version](https://www.arrikto.com/wp-content/uploads/2019/06/minikf-latest-version.svg
-"MiniKF latest version")
 
-A production-ready, full-fledged, local Kubeflow deployment that
+MiniKF is a production-ready, full-fledged, local Kubeflow deployment that
 installs in minutes.
 
 MiniKF is a fast and easy way to get started with Kubeflow. With just a
```
Comment (8bitmp3) [inline content/en/docs/started/workstation/getting-started-minikf.md:10]: Suggestion: 

- Merge these, then both sentences don't have to start with "MiniKF.... MiniKF.."

```suggestion
MiniKF is a production-ready, full-fledged, local Kubeflow deployment that
installs in minutes. It is a fast and easy way to get started with Kubeflow. With just a
```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
@jbottum Hi, a few suggestions for this PR:

- Improve the title for better discoverability. Also, new users should be able to understand what this guide does quicker, hopefully, if we tweak it a little.

cc @alfsuse 

```diff
- title = "MiniKF"		
+ title = "Deploy Kubeflow locally with MiniKF"
```

Also, maybe we should merge lines 17 and 19 - then both sentences don't have to start with "MiniKF.... MiniKF..."

```
MiniKF is a production-ready, full-fledged, local Kubeflow deployment that
installs in minutes. It is a fast and easy way to get started with Kubeflow. With just a
...
```

File: content/en/docs/started/workstation/getting-started-minikf.md [hunk 2]
```diff
@@ -3,17 +3,8 @@ title = "MiniKF"
 description = "A fast and easy way to deploy Kubeflow on your laptop"
 weight = 35
                     
-+++
```
Comment (alfsuse) [inline content/en/docs/started/workstation/getting-started-minikf.md:6]: Hi, @jbottum also you have to put back this line. the '+++' is needed for the title and without that the deployment will fail.

**Comment by 8bitmp3** [comment]
Hi @jbottum let us know what you think and if you have an update. @alfsuse and I are here to help. Cheers! 
cc @RFMVasconcelos

**Comment by jbottum** [comment]
Hi @8bitmp3 , thanks for the improvements.   The MiniKF PM asked me to close this and will open another issue in a few days.

**Comment by jbottum** [comment]
Let's close this unmerged.   the MiniKF PM will open another issue for the update in a few days.   thanks for your help.

**Pull request closed without merging**